pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -87,6 +87,7 @@
 
   GROUP_BY_SIZE("queries", false),
   TOTAL_SERVER_RESPONSE_SIZE("queries", false),
+  QUERY_COLUMN_NAME_MISMATCH("queries", false),
```

Comment:
It is more of a case of column name in query missing from schema, than a column name mismatch, right?

Reply:
How about we rename it to `QUERY_COLUMN_NAME_MISSING_IN_SCHEMA`?

Reply:
In other places (DataTable), you have used INVALID_COLUMN, so may be `INVALID_COLUMN_NAME_IN_QUERY`?

Reply:
Updated the meter name